ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -122,6 +122,19 @@ CREATE TABLE participant_command_completions (
     command_id VARCHAR NOT NULL,
     -- The transaction ID is `NULL` for rejected transactions.
     transaction_id VARCHAR,
+    -- The application ID has to be provided by the application.
+    application_id VARCHAR NOT NULL,
```

Comment:
There is already an `application_id` above.

Reply:
8079673